sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/GetAppServiceDataResponse.java

```diff
@@ -0,0 +1,48 @@
+package com.smartdevicelink.proxy.rpc;
+
+import android.support.annotation.NonNull;
+
+import com.smartdevicelink.protocol.enums.FunctionID;
+import com.smartdevicelink.proxy.RPCResponse;
+
+import java.util.Hashtable;
+
+/**
+ * This response includes the data that is requested from the specific service
+ */
+public class GetAppServiceDataResponse extends RPCResponse {
+
+	public static final String KEY_SERVICE_DATA = "serviceData";
+
+	// Constructors
+
+	public GetAppServiceDataResponse(){
+		super(FunctionID.GET_APP_SERVICE_DATA.toString());
+	}
+
+	public GetAppServiceDataResponse(Hashtable<String, Object> hash){
+		super(hash);
+	}
+
+	public GetAppServiceDataResponse(@NonNull AppServiceData serviceData){
```

Comment:
This constructor should have all mandatory param including success and resultCode

Reply:
Fixed